classes/emailTemplate/DAO.php

```diff
@@ -153,9 +153,18 @@ public function fromRow(object $row): EmailTemplate
         /** @var EmailTemplate $emailTemplate */
         $emailTemplate = parent::fromRow($row);
         $schema = $this->schemaService->get($this->schema);
+        $contextDao = Application::getContextDAO();
+
+        $supportedLocalesJson = $row->context_id === PKPApplication::CONTEXT_SITE ?
+            DB::table('site')->first()->supported_locales :
+            DB::table($contextDao->settingsTableName)
+                ->where($contextDao->primaryKeyColumn, $row->context_id)
+                ->where('setting_name', 'supportedLocales')
+                ->value('setting_value');
 
         $rows = DB::table($this->defaultTable)
             ->where('email_key', '=', $emailTemplate->getData('key'))
+            ->whereIn('locale', json_decode($supportedLocalesJson))
```

Comment:
This caused a fatal error for me -- `json_decode` returns a `stdClass`, and `whereIn` needs an array.

Reply:
Hmm, that's strange. Is `supportedLocales` an associative array in your case? I'd expect it to be in the format `["en_US","fr_CA"]`. I can enforce it to return an array but I'm wondering if it's a revenge of https://github.com/pkp/pkp-lib/issues/6757

Reply:
You're probably right -- but as a precaution it might be a good idea to set `$associative` to `true` in the call to `json_decode`.

Reply:
...aaaand I see you've already done that :+1: